Source/ToolCore/Import/OpenAssetImporter.cpp

```diff
@@ -860,6 +866,70 @@ void OpenAssetImporter::CollectAnimations(OutModel* model)
     /// \todo Vertex morphs are ignored for now
 }
 
+void OpenAssetImporter::ApplyFlag(int flag, bool active)
+{
+    aiFlagsDefault_ &= ~flag;
+    if (active)
+        aiFlagsDefault_ |= flag;
+}
+
+void OpenAssetImporter::SetOveriddenFlags(VariantMap aiFlagParameters)
```

Comment:
Can this be (const VariantMap& aiFlagParameters, otherwise making a copy when passing to the method